tools/proto-compiler/src/cmd/compile.rs

```diff
@@ -147,7 +147,7 @@ impl CompileCmd {
             .server_mod_attribute(".", r#"#[cfg(feature = "server")]"#)
             .out_dir(out_dir)
             .file_descriptor_set_path(out_dir.join("proto_descriptor.bin"))
-            .extern_path(".tendermint", "::tendermint_proto")
+            .extern_path(".tendermint", "::cometbft_proto")
```

Comment:
Simply doing this change is not enough. We need to select between the different options within `cometbft_proto`.

```
error[E0412]: cannot find type `SignedHeader` in module `cometbft_proto::types`
   --> src/prost/ibc.lightclients.tendermint.v1.rs:128:72
    |
128 |     pub signed_header: ::core::option::Option<::cometbft_proto::types::SignedHeader>,
    |                                                                        ^^^^^^^^^^^^ not found in `cometbft_proto::types`
    |
help: consider importing one of these items
   --> src/lib.rs:373:17
    |
373 +                 use cometbft_proto::types::v1::SignedHeader;
   --> |src/lib.rs:373:17
    |
373 +                 use cometbft_proto::types::v1beta1::SignedHeader;
```

cc @mzabaluev any thoughts?

Reply:
It seems ibc-proto is still using the `tendermint.*` packages, not `cometbft.*`. To be able to use the modules from `cometbft_proto`, please switch to importing `cometbft` packages on the protobuf level. There you'll need to specify the versioned paths to proto packages you intend to be using with your revision of IBC.

Reply:
Trouble is, the `tendermint` protos are still bundled with the SDK repository checkout and they may be outdated.
So this may be blocked on the SDK updating their proto files.

Reply:
On the other hand, the sole reason for the SDK checkout seems to be to import "tendermint/crypto/proof.proto". So I'll try replacing that checkout completely with cometbft protos, let's see if difficulties arise.

Reply:
> the sole reason for the SDK checkout seems to be to import "tendermint/crypto/proof.proto".

That wasn't true, of course: cosmos and ICS packages are compiled as part of this library, and `tendermint` is used all over the place in those protos. So this is blocked on upstream protos converting to `cometbft` first.